docs/TheoryDeepPixels.rst

```diff
@@ -0,0 +1,491 @@
+======================
+Theory of Deep Samples
+======================
+
+
+This document derives the techniques for splitting and combining two non-solid samples of
```

Comment:
Make a reference to the Interpreting doc: `Interpreting OpenEXR Deep Pixels`_.

Reply:
The other cross-document links don't seem to work for me (e.g. the link from OpenEXRFileLayout to TechnicalIntroduction didn't work) but using `:doc:` seems OK